components/config/README.md

```diff
@@ -0,0 +1,68 @@
+# <img src="torch-logo.png" alt="Torch Logo" align="right">Torch - Using Laravel's Illuminate Components Independently
+
+Torch is a project to provide instructions and examples for using Illuminate components as standalone components in non-Laravel applications. The current `master` branch shows how to use Illuminate's `5.1` components.
+
+**Note**: If you are working with an older project, you might have more success using the [4.2 components](https://github.com/mattstauffer/torch/tree/4.2).
+
+## Config Component
+
+The config component tries to demonstrate how we can use the [Laravel Config](https://laravel.com/docs/5.4/configuration/) package to store our configs in a array and display the same.
```

Comment:
Maybe update the documentation link to the 5.5 version?

Reply:
I thinks this components are related to 5.4 only . I need to check if they work with 5.5 too.

Reply:
These components are 5.1. :)

Reply:
you are right i've mistake 4.4 with 5.4 shame on me 👎